src/Game.js

```diff
@@ -0,0 +1,41 @@
+export default class Game {
+    constructor() {
+        // this.alive = true;
+        this.player1;
+        this.player2;
+        this.wordToGuess = '';
+        this.attemptedGuesses = []; // All of these will be displayed as guesses
+        this.maxWrongAttempts = 6;
+        this.wrongGuesses = 0;
+        this.correctGuesses = []; // These will be displayed as correct guesses only
+    }
+    reviewAttempt(guess) {
+        this.attemptedGuesses.push(guess);
+        if (!this.wordToGuess == guess || !this.wordToGuess.includes(guess)) {
+            this.wrongGuesses += 1;
+            this.wrongAttempt();
+        } else {
+            this.correctGuesses.push(guess);
+            this.correctAttempt();
+        }
+    }
+    wrongAttempt() {
+        if (this.wrongGuesses >= this.maxWrongAttempts) {
+            alert('The man is dead');
+            location.reload();
```

Comment:
What is `location.reload()`?

Reply:
reloads the page ;) Nice way to start from scratch. This can always be changed in the future but it works for now.

Reply:
O.O Wow, I did not know that! Is `location` a module of our browser? This is awesome, if so!